machinery/gateway_api_topology_test.go

```diff
@@ -20,6 +20,10 @@ import (
 // This results in a topology with the following scheme:
 //
 //	GatewayClass -> Gateway -> HTTPRoute -> Service
+//							∟> GRPCRoute ⤴
+//							∟> TCPRoute  ⤴
+//							∟> TLSRoute  ⤴
+//							∟> UDPRoute  ⤴
```

Comment:
I guess some white spaces have been converted to tabs here.

```suggestion
//	                        ∟> GRPCRoute ⤴
//	                        ∟> TCPRoute  ⤴
//	                        ∟> TLSRoute  ⤴
//	                        ∟> UDPRoute  ⤴
```